### ISSUE TITLE
Add translations info to developer setup doc

### ISSUE THREAD

**Issue by antsar** [freedomofpress/securedrop#3433]
Following the [Setting up the development environment](https://docs.securedrop.org/en/release-0.7/development/setup_development.html) doc produces a SecureDrop instance with nonfunctional translations. Selecting a language has no effect - the UI is still rendered in English. [Compiling the translations](https://docs.securedrop.org/en/release-0.7/development/i18n.html#compiling-translations) fixes this issue.

Extend the Setting up the [Setting up the development environment](https://docs.securedrop.org/en/release-0.7/development/setup_development.html) doc to indicate that translations must be compiled in order for i18n to work.

## User Stories

As a new SecureDrop contributor, I expected that translations would be functional in the developer environment.

**Comment by dachary** [comment]
I tried to reproduce the problem as follows:

* make -C securedrop dev
* firefox http://localhost:8080/?l=fr_FR

but I see the page displaying in French. Would you be so kind as to detail the steps I should take to reproduce the problem?

**Comment by dachary** [comment]
You are absolutely correct, good catch! Hopefully https://github.com/freedomofpress/securedrop/pull/3436 fixes it.
